janno_details.md

```diff
@@ -174,14 +175,14 @@ The `Contamination_Note` column is a free text field to add additional informati
 
 ### Context information
 
-The `Genetic_Source_Accession_IDs` column was introduced to link the derived genotype data in Poseidon with the raw sequencing data typically uploaded to archives like the ENA [@Burgin2022](https://doi.org/10.1093/nar/gkac1051) or SRA [@Katz2021](https://doi.org/10.1093/nar/gkab1053). There projects or even individual samples are given clear identifiers: Accession IDs. This janno column is supposed to store one or multiple of these Accessions IDs for each individual/sample in Poseidon. If multiple are entered, then they should be arranged by descending specificity from left to right (e.g. project id > sample id > sequencing run id).
+The `Genetic_Source_Accession_IDs` column was introduced to link the derived genotype data in Poseidon with the raw sequencing data typically uploaded to archives like the ENA [@Burgin2022](https://doi.org/10.1093/nar/gkac1051) or SRA [@Katz2021](https://doi.org/10.1093/nar/gkab1053). There, projects and individual samples are given clear unique identifiers: Accession IDs. This janno column is supposed to store one or multiple of these Accessions IDs for each individual/sample in Poseidon. If multiple are entered, then they should be arranged by descending specificity from left to right (e.g. project id > sample id > sequencing run id).
 
-The `Primary_Contact` column is a free form text field that stores the name of the main or the corresponding author of the respective paper for published data.
+The `Primary_Contact` column is a free-form text field that stores the name of the main or the corresponding author of the respective paper for published data.
 
 The `Publication` column holds either the value `unpublished` for (yet) unpublished samples or -- for published data -- one or multiple citation-keys of the form `AuthorJournalYear` without any spaces or special characters. These keys have to be identical to the [BibTeX](http://www.bibtex.org) citation-keys identifying the respective entries in the `.bib` file of the package. BibTeX is a file format to store bibliographic information, where each entry (article, book, website, ...) is defined by a series of parameters (authors, year of publication, journal, ...). Here's an example `.bib` file with two entries for [@Cassidy2015](https://doi.org/10.1073/pnas.1518445113) and [@Feldman2019](https://doi.org/10.1126/sciadv.aax0061):
 
 ```default
-@article{Cassidy2015,
+@article{CassidyPNAS2015,
```

Comment:
Updated these keys to match the text.

Reply:
OK!

Reply:
Wait. The text has `@Cassidy2015` and `@Feldman2019` above and `CassidyPNAS2015;FeldmanScienceAdvances2019` below. Maybe we rather adjust that.

Reply:
I believe @Cassidy2015 and @Feldman2019 refer to the bibtex of the manuscript, while `CassidyPNAS2015;FeldmanScienceAdvances2019` to the bibtex we're showcasing (this is clearer in the PDF version).
The poseidon schema is clear that `CassidyPNAS2015;FeldmanScienceAdvances2019` is the right formatting for Poseidon (`AuthorJournalYear`), so i believe editing the shown bibtex is the right way to go.

Reply:
OK - sounds good to me :+1: